docs/expressions.md

```diff
@@ -330,6 +345,20 @@ Remove any values lower than or equal to number from a series. Will error if thi
 
 Remove any NaN or Inf values from a series. Will error if this operation results in an empty series.
 
+## epoch()
+
+Returns the Unix epoch in seconds of the expression start time (scalar).
+
+## filter(series, number)
+
+Returns all results in series that are a subset of anything in number, or
```

Comment:
Filters the first argument, `series` based on the second argument, `number`. Only the groups in the first argument that are a subset of the second argument are returned. So this allows you to filter out series based on the same or a different series.

Does that help at all @captncraig ?